backend/arm64/proc.ml

```diff
@@ -275,7 +275,7 @@ let destroyed_at_oper = function
       all_phys_regs
   | Iop(Iextcall { alloc = false; }) ->
       destroyed_at_c_call
-  | Iop(Ialloc _) ->
+  | Iop(Ialloc _) | Iop(Ipoll _) ->
```

Comment:
Shouldn't we have a `Poll_and_jump`
in `destroyed_at_terminator` below?
(cannot add the comment there...)